libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -173,6 +173,18 @@ const decoratePromo = (elem, index) => {
     elem.classList.add('feds-promo--dark');
   }
 
+  if (!isImageOnly) {
+    const imageElemMaybe = imageElem && elem.removeChild(elem.querySelector('.feds-promo-image'));
+    const innerContainer = document.createElement('div');
+    innerContainer.classList.add('feds-promo-inner-container');
+
+    while (elem.firstChild) innerContainer.appendChild(elem.firstChild);
```

Comment:
is there a way to append all elements at once?

Reply:
Solved